general/administration/installing.md

```diff
@@ -63,25 +63,38 @@ An Unraid Docker template is available in the repository.
 
 A community project to deploy Jellyfin on Kubernetes-based platforms exists [at their repository](https://github.com/home-cluster/jellyfin-openshift). Any issues or feature requests related to deployment on Kubernetes-based platforms should be filed there.
 
-## Portable Binaries
+## Windows (x64)
 
-Portable binary packages containing a compiled Jellyfin instance are available for multiple platforms. For all portable binaries, extract the relevant archive to a directory and launch the binary (`jellyfin` on Linux/MacOS, `jellyfin.exe` on Windows).
+Windows installers and builds in ZIP archive format are available [here](https://repo.jellyfin.org/releases/server/windows).
 
-### Windows (x64/x86)
+[!WARNING]
+> If you installed a version prior to 10.4.0 using a powershell script, you will need to manually remove the service and uninstall the server. Also one might need to move the data files to the correct location, or point the installer at the old location.
 
-Windows builds in ZIP archive format are available [here](https://repo.jellyfin.org/releases/server/windows).
+### Install using installer
 
-### Install via script
```

Comment:
I feel that this section should be retained for those who would want to work with pre-10.4.0 releases